bin/shell.ps1

```diff
@@ -33,6 +33,7 @@ $env:PATH += ";${elxInstallPath}\bin"
 $env:PATH += ";${vcpkgBase}\bin"
 $env:PATH += ";C:\Program Files (x86)\WiX Toolset v3.11\bin"
 $env:PATH += ";C:\tools\${java11Build}\bin"
+$env:PATH += ";C:\tools\msys64\ucrt64\bin"
```

Comment:
Short question, where is this path set?

Reply:
This is set by the msys2 install after chocolatey installs it. Here we're just making gcc.exe and ld.exe available to the build environment so when we `make` the `couch_quickjs` app we don't have to hard-code `c:\tools\...\gcc.exe` in there and can just use `gcc.exe ...`

Reply:
This is required for QuickJS https://github.com/apache/couchdb/pull/4627 to build on Windows

Reply:
I had tired to uninstalled and clean all the msys2 bits, then re-install and it still went to `c:\tools\msys64` so at least on my laptop seems to end up in `c:\tools` I can't see where it's configured to do so...